### PR TITLE
Add SplitDebtCommand

### PR DESCRIPTION
**Pull request by WR3nd3**
SplitDebtCommand and SplitDebtCommandParser classes are added.
An example of the command is splitdebt 1 2 0 d/pizza m/3.33
Index '0' refers to the user. Repeated indexes are treated as a single index and the '0' index cannot be by itself.
Indexes must be separated by a single space.
Money is rounded up to the closest cent after division.

UG/DG are not updated.

### CODE REVIEW

**Review by waynezsy — COMMENTED** [review]
LGTM, but will definitely break once Time/Date is implemented into Debt.
Would recommend holding off merging this until #129 is merged.

**Review by lyuting47 — DISMISSED** [review]
LGTM but need to resolve conflicts first

**Review by lyuting47 — APPROVED** [review]
LGTM

**Pull request merged**
